ui/src/groups/GroupAdmin/GroupInfoEditor.tsx

```diff
@@ -40,17 +40,18 @@ export default function GroupInfoEditor({ title }: ViewProps) {
     defaultValues: {
       ...emptyMeta,
       ...group?.meta,
-      privacy: (group && getGroupPrivacy(group.cordon)) || 'public',
+      privacy: group ? getPrivacyFromGroup(group) : undefined,
     },
   });
 
-  useEffect(() => {
-    form.reset({
-      ...emptyMeta,
-      ...group?.meta,
-      privacy: (group && getGroupPrivacy(group.cordon)) || 'public',
-    });
-  }, [group, form]);
+  // TODO: is this still needed? don't want to get Chesterton's Fence'd
```

Comment:
I believe we still need this we need to reload the form with the correct privacy if it's not immediately available on first mount.